config/src/test-support/java/org/hyperledger/besu/config/StubGenesisConfigOptions.java

```diff
@@ -181,6 +181,11 @@ public OptionalInt getEvmStackSize() {
     return builder.build();
   }
 
+  @Override
+  public CustomForksConfigOptions getCustomForks() {
+    return null;
```

Comment:
Should we update this to return a `CustomForksConfigOptions` instance empty instead of null?

Reply:
yeah, we should...